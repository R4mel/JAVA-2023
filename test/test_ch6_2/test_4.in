1
8
s1

15
1
2
,
2
the
2
of
2
is
2
much
0
0
0
0
