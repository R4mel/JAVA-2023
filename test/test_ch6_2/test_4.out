
******* Main Menu ********
* 0.exit 1.PersonManager *
* 2.ch2 3.ch3 4.ch5      *
**************************
menu item? 1
[0] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2)                                            =
================================================================
menu item? 8
user name: s1
password: 

++++++++++++++++++++++ Current User Menu ++++++++++++++++++++++++
+ 0.logout 1.display 2.getter 3.setter 4.copy 5.whatAreYouDoing +
+ 6.isSame 7.update 8.chPasswd(4_2) 9.chSmartPhone(5_3)         +
+ 10.clone(5_3) 11.calc(5_3) 12.phoneCall(5_3) 13.chWeight(6_1) +
+ 14.calcString(6_2) 15.memo(6_2)                               +
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
menu item? 15

+++++++++++++++++++++ Memo Management Menu +++++++++++++++++++
+ 0.exit 1.display 2.find 3.findReplace 4.compare 5.dispByLn +
+ 6.delLn 7.replLn 8.scrollUp 9.scrollDown 10.inputMemo      +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
menu item? 1
------- Memo -------
The Last of the Mohicans
James Fenimore Cooper
Author's Introduction
It is believed that the scene of this tale, and most of the information
necessary to understand its allusions, are rendered sufficiently 
obvious to the reader in the text itself, or in the accompanying notes.
Still there is so much obscurity in the Indian traditions, and so much
confusion in the Indian names, as to render some explanation useful.
Few men exhibit greater diversity, or, if we may so express it, 
greater antithesis of character, 
than the native warrior of North America.
--------------------

+++++++++++++++++++++ Memo Management Menu +++++++++++++++++++
+ 0.exit 1.display 2.find 3.findReplace 4.compare 5.dispByLn +
+ 6.delLn 7.replLn 8.scrollUp 9.scrollDown 10.inputMemo      +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
menu item? 2
word to find? ,
--------------------
0 words found

+++++++++++++++++++++ Memo Management Menu +++++++++++++++++++
+ 0.exit 1.display 2.find 3.findReplace 4.compare 5.dispByLn +
+ 6.delLn 7.replLn 8.scrollUp 9.scrollDown 10.inputMemo      +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
menu item? 2
word to find? the
[0] The Last of the Mohicans
[3] It is believed that the scene of this tale, and most of the information
[5] obvious to the reader in the text itself, or in the accompanying notes.
[6] Still there is so much obscurity in the Indian traditions, and so much
[7] confusion in the Indian names, as to render some explanation useful.
[10] than the native warrior of North America.
--------------------
9 words found

+++++++++++++++++++++ Memo Management Menu +++++++++++++++++++
+ 0.exit 1.display 2.find 3.findReplace 4.compare 5.dispByLn +
+ 6.delLn 7.replLn 8.scrollUp 9.scrollDown 10.inputMemo      +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
menu item? 2
word to find? of
[0] The Last of the Mohicans
[3] It is believed that the scene of this tale, and most of the information
[9] greater antithesis of character, 
[10] than the native warrior of North America.
--------------------
5 words found

+++++++++++++++++++++ Memo Management Menu +++++++++++++++++++
+ 0.exit 1.display 2.find 3.findReplace 4.compare 5.dispByLn +
+ 6.delLn 7.replLn 8.scrollUp 9.scrollDown 10.inputMemo      +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
menu item? 2
word to find? is
[3] It is believed that the scene of this tale, and most of the information
[6] Still there is so much obscurity in the Indian traditions, and so much
--------------------
2 words found

+++++++++++++++++++++ Memo Management Menu +++++++++++++++++++
+ 0.exit 1.display 2.find 3.findReplace 4.compare 5.dispByLn +
+ 6.delLn 7.replLn 8.scrollUp 9.scrollDown 10.inputMemo      +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
menu item? 2
word to find? much
[6] Still there is so much obscurity in the Indian traditions, and so much
--------------------
2 words found

+++++++++++++++++++++ Memo Management Menu +++++++++++++++++++
+ 0.exit 1.display 2.find 3.findReplace 4.compare 5.dispByLn +
+ 6.delLn 7.replLn 8.scrollUp 9.scrollDown 10.inputMemo      +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
menu item? 0

++++++++++++++++++++++ Current User Menu ++++++++++++++++++++++++
+ 0.logout 1.display 2.getter 3.setter 4.copy 5.whatAreYouDoing +
+ 6.isSame 7.update 8.chPasswd(4_2) 9.chSmartPhone(5_3)         +
+ 10.clone(5_3) 11.calc(5_3) 12.phoneCall(5_3) 13.chWeight(6_1) +
+ 14.calcString(6_2) 15.memo(6_2)                               +
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
menu item? 0

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2)                                            =
================================================================
menu item? 0

******* Main Menu ********
* 0.exit 1.PersonManager *
* 2.ch2 3.ch3 4.ch5      *
**************************
menu item? 0

Good bye!!
